OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -190,14 +190,21 @@ Action InitDisplayPanel(Widget panel)
 
 			var frameLimitTextfield = panel.Get<TextFieldWidget>("FRAME_LIMIT_TEXTFIELD");
 			frameLimitTextfield.Text = ds.MaxFramerate.ToString();
-			frameLimitTextfield.OnLoseFocus = () =>
+			frameLimitTextfield.OnEnterKey = () =>
```

Comment:
This has the same issue as the lobby.  If you unfocus using the mouse the widget text will no longer match the underlying data.